src/components/AccordionItem/AccordionItem.js

```diff
@@ -133,12 +135,22 @@ export default class AccordionItem extends Component {
           aria-expanded={this.state.open}
           className={`${prefix}--accordion__heading`}
           onClick={this.handleHeadingClick}>
-          <Icon
-            className={`${prefix}--accordion__arrow`}
-            icon={iconChevronRight}
-            description={iconDescription}
-            role={null} // eslint-disable-line jsx-a11y/aria-role
-          />
+          {componentsX ? (
+            <ChevronRight16
+              className={`${prefix}--accordion__arrow`}
+              icon={iconChevronRight}
+              aria-label={iconDescription}
+              role={null} // eslint-disable-line jsx-a11y/aria-role
```

Comment:
What's the rationale again for using `null` for role? cc @emyarod

Reply:
actually that might be redundant for the new icons so we can probably remove that line